src/bookshelf/rebar.config

```diff
@@ -10,26 +10,41 @@
     %% lager has to come first since we use its parse transform
     {lager, ".*",
         {git, "https://github.com/erlang-lager/lager",              {branch, "master"}}},
+    % this is here because of license scout issues
+    {base16, ".*",
+        {git, "https://github.com/goj/base16",                      {tag,    "1.0.0"}}},
```

Comment:
Could you please let us know where are we using this library in chef-server code.

Reply:
@sreepuramsudheer Is trying to test by removing newly added deps for licence_scout to understand the actual issue.

Reply:
license_scout changes have been removed from this PR, and put in the final PR for the Erlang upgrade.